{
	"id": "ALSA-2022:7730",
	"advisories": [
		{
			"id": "ALSA-2022:7730",
			"title": "Moderate: libldb security, bug fix, and enhancement update",
			"description": "The libldb packages provide an extensible library that implements an LDAP-like API to access remote LDAP servers, or use local TDB databases.\n\nThe following packages have been upgraded to a later upstream version: libldb (2.5.2). (BZ#2077484)\n\nSecurity Fix(es):\n\n* samba: AD users can induce a use-after-free in the server process with an LDAP add or modify request (CVE-2022-32746)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n\nAdditional Changes:\n\nFor detailed information on changes in this release, see the AlmaLinux Release Notes linked from the References section.",
			"severity": [
				{
					"type": "vendor",
					"source": "errata.almalinux.org",
					"vendor": "Moderate"
				}
			],
			"references": [
				{
					"source": "errata.almalinux.org",
					"url": "https://access.redhat.com/security/cve/CVE-2022-32746"
				},
				{
					"source": "errata.almalinux.org",
					"url": "https://bugzilla.redhat.com/2108215"
				},
				{
					"source": "errata.almalinux.org",
					"url": "https://errata.almalinux.org/8/ALSA-2022-7730.html"
				},
				{
					"source": "errata.almalinux.org",
					"url": "https://access.redhat.com/errata/RHSA-2022:7730"
				}
			],
			"published": "2022-11-08T00:00:00Z",
			"modified": "2022-11-14T10:54:12Z"
		}
	],
	"vulnerabilities": [
		{
			"id": "CVE-2022-32746",
			"references": [
				{
					"source": "errata.almalinux.org",
					"url": "https://access.redhat.com/security/cve/CVE-2022-32746"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "python3-ldb",
				"architectures": [
					"i686",
					"x86_64",
					"aarch64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:2.5.2-2.el8"
					}
				],
				"fixed": [
					"0:2.5.2-2.el8"
				]
			}
		},
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "libldb",
				"architectures": [
					"i686",
					"x86_64",
					"aarch64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:2.5.2-2.el8"
					}
				],
				"fixed": [
					"0:2.5.2-2.el8"
				]
			}
		},
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "python3-ldb-devel",
				"architectures": [
					"i686",
					"aarch64",
					"x86_64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:2.5.2-2.el8"
					}
				],
				"fixed": [
					"0:2.5.2-2.el8"
				]
			}
		},
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "python-ldb-devel-common",
				"architectures": [
					"i686",
					"aarch64",
					"x86_64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:2.5.2-2.el8"
					}
				],
				"fixed": [
					"0:2.5.2-2.el8"
				]
			}
		},
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "ldb-tools",
				"architectures": [
					"x86_64",
					"aarch64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:2.5.2-2.el8"
					}
				],
				"fixed": [
					"0:2.5.2-2.el8"
				]
			}
		},
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "libldb-devel",
				"architectures": [
					"i686",
					"x86_64",
					"aarch64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:2.5.2-2.el8"
					}
				],
				"fixed": [
					"0:2.5.2-2.el8"
				]
			}
		}
	],
	"data_source": "alma-errata"
}
